addons/website/static/src/scss/website.scss

```diff
@@ -378,6 +474,16 @@ font[class*='bg-'] {
 
 // Buttons
 .btn {
+    border-style: o-website-value('btn-border-style');
+
+    &.btn-xs, &.btn-sm {
+        border-width: o-website-value('btn-border-width-sm');
+        border-style: o-website-value('btn-border-style-sm');
+    }
+    &.btn-lg {
+        border-width: o-website-value('btn-border-width-lg');
+        border-style: o-website-value('btn-border-style-lg');
+    }
```

Comment:
@qsm-odoo  & @bso-odoo 
 > Should the "style" be in a collapse? Hard to find, no?
good point, let's have like before, two distinct options without 
![image](https://github.com/odoo/odoo/assets/75257428/85bb5854-58c7-416f-84d9-577790b029e0)

 > Should we suggest the border configuration always/at all? 
Let's have the following: 
1. only display border width if style == outline
2. prevent user from choosing a border style
3. have only one border width, wathever the button size is. 
![image](https://github.com/odoo/odoo/assets/75257428/9f6f2e4e-2b7f-46b3-9ca3-abd00c0a9b4d)

Reply:
@pierre-pde 

> only display border width if style == outline

You mean if style primary **or** style secondary is oultine? And I guess we should display it below secondary instead of primary then?

> have only one border width, wathever the button size is.

Same question I guess... :)

Reply:
> You mean if style primary or style secondary is oultine? And I guess we should display it below secondary instead of primary then?
Yeah, IMO, you should be able to choose two different border width if you select "outline" twice
However, you shouldn't be able to choose one border width per button size, it's always the same.

Reply:
@pierre-pde Nested one "Border Width" below the ouline style of both primary & secondary.
Renamed border radius to "Round Corners".
Removed general border widths and styles (all sizes).
<https://github.com/odoo/odoo/compare/8064006f1b8e1661217fd9c36fbfde177097a26f..e74ae4c109c6fd90ebd253c5d1ca9e9ad490e2e5>